src/style.css

```
@@ -0,0 +1,91 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir" ;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 700;
+}
+
+html input {
+  font-family: Avenir, Helvetica, sans-serif;
+}
+
+.input {
+  font-style: normal;
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 300;
+  height: 70px;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  padding-left: 62px;
+  color: #3d4e61;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.icon-big {
+  position: absolute;
+  margin: 25px 26px 0;
+}
+
+.small-form {
+  margin-top: 20px;
+}
```

```suggestion
.form {
  margin-top: 20px;
}

.icon-big {
  position: absolute;
  margin: 25px 26px 0;
}

.small-form {
  margin-top: 20px;
}
```
form and small-form have the same styles